CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 - Added handling for `object_name` and `extra_keywords` to `sum_vis` and `diff_vis` methods and added the `override_params` option to override other parameters.
+- Added handling for `extra_keywords` in UVFlag objects, including read/write to/from files.
```

Comment:
Please add your changes in a section to the top of the section. We use reverse chronological order for each subsection (e.g. newest first).

Reply:
Should be fixed now.